nncf/tensorflow/graph/sequential_model.py

```diff
@@ -0,0 +1,36 @@
+"""
```

Comment:
Why is this file here? It seems like rebase error.

Reply:
Yes, seems like some error while rebasing. Removed it.